interface-definitions/interfaces_vxlan.xml.in

```diff
@@ -117,15 +117,35 @@
                 <format>u32:0-4094</format>
                 <description>Virtual Local Area Network (VLAN) ID</description>
               </valueHelp>
+              <valueHelp>
+                <format>&lt;start-end&gt;</format>
+                <description>VLAN ids range (use '-' as delimiter)</description>
+              </valueHelp>
               <constraint>
-                <validator name="numeric" argument="--range 0-4094"/>
+                <validator name="numeric" argument="--allow-range --range 0-4094 "/>
               </constraint>
-              <constraintErrorMessage>VLAN ID must be between 0 and 4094</constraintErrorMessage>
+              <constraintErrorMessage>Not a valid VLAN ID or range</constraintErrorMessage>
```

Comment:
Please change to: `Not a valid VLAN ID or range, VLAN ID must be between 0 and 4094` to give a proper hint to the user.

Reply:
Other then that it looks good to me.